.github/workflows/mobilecoin-dev-cd.yaml

```
@@ -106,7 +106,7 @@ jobs:
         VIEW_ENCLAVE_PRIVKEY: ${{ env.ENCLAVE_SIGNING_KEY_PATH }}
         INGEST_ENCLAVE_PRIVKEY: ${{ env.ENCLAVE_SIGNING_KEY_PATH }}
       run: |
-        ls -al "${ENCLAVE_SIGNING_KEY_PATH}"
+        git config --global --add safe.directory '*'
```

This incantation seems worth documenting
```suggestion
        # Workaround for https://github.com/actions/checkout/issues/766
        git config --global --add safe.directory '*'
```